src/NzbDrone.Common.Test/Http/HttpUriFixture.cs

```diff
@@ -0,0 +1,84 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using NzbDrone.Common.Http;
+using NzbDrone.Test.Common;
+
+namespace NzbDrone.Common.Test.Http
+{
+    public class HttpUriFixture : TestBase
+    {
+        [TestCase("", "", "")]
+        [TestCase("/", "", "/")]
+        [TestCase("base", "", "base")]
+        [TestCase("/base", "", "/base")]
+        [TestCase("/base/", "", "/base/")]
+        [TestCase("", "relative", "relative")]
+        [TestCase("", "/relative", "/relative")]
+        [TestCase("/", "relative", "/relative")]
+        [TestCase("/", "/relative", "/relative")]
+        [TestCase("base", "relative", "relative")]
+        [TestCase("base", "/relative", "/relative")]
+        [TestCase("/base", "relative", "/relative")]
```

Comment:
In Uri Combine it does. truncates the 'file' part. Check it out in your browser:
<a href="1024">test 1</a> `<a href="1024">test 1</a>`
<a href="/1024">test 2</a> `<a href="/1024">test 2</a>`